### DISCUSSION TITLE
Autoremove DDSN records

### DISCUSSION THREAD

**Discussion by PeterKnotek** [TechnitiumSoftware/DnsServer#934] [General]
Host added via DDNS 
   - domain is  example XYZ.SK 
   - created via ISC DHCP 
   -  TTL is example 3600sec
- are  removed automatically ??? 
- or will they remain permanently created?? 

Well thank you

**Comment by ShreyasZare** [comment]
Thanks for asking. The Dynamic Updates protocol allows only adding/updating/deleting records. There is no provision to set an expiry for a record. The DHCP server is thus responsible for deleting the DNS record when the DHCP lease expires.
